dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -1882,6 +1859,29 @@
       },
       "x-ms-client-name": "AssignedProjectDeploymentsMetadata"
     },
+    "ConversationalAnalysisAuthoringAssignedResourceDeploymentsMetadata": {
+      "description": "Represents the metadata for deployments assigned to a resource.",
+      "required": [
+        "value",
+        "nextLink"
```

Comment:
Why is this required? The last page shouldn't have it, which also means you shouldn't use `x-nullable: true`. Ideally, we should never see x-nullable.

Reply:
The behavior we have in our previous APIs including the GA 2022-05-01 is that it is required and nullable, so for the last page it will show with a value of null. I am not sure if we can change it now or that will be considered a breaking change?